### PR TITLE
Use getters to remove parens

### LINKED ISSUE #18: Simplify method chain
The current API calls are like this:

image

Would it be better to have something simpler.

```js
Tw().bgfff.lt960FlexNone.lt960WFull.$()
```

The implementation could be done with Proxy.

### PR DESCRIPTION
**Pull request by stephenh**
Fixes #18

### CODE REVIEW

File: convert/src/file.ts [hunk 1]
```diff
@@ -7,24 +7,23 @@ const base: { open: string, close: string } = { open: "", close: "" };
 const twClass = "Tailwind";
 
 base.open = `// http://github.com/dvkndn/typed-tailwind
-export const Tw = (): ${twClass} => new ${twClass}();
+export const Tw = new ${twClass}();
```
Comment (stephenh) [inline convert/src/file.ts:10]: This is safe to create a new `Tw` immediately b/c all of the `.foo()` methods make copies/new `Tailwind` instances, i.e. each `Tailwind` instance is immutable.

**Comment by dvkndn** [comment]
Thank YOU! I'll review this ASAP but I think maybe this weekend soonest

**Comment by adamgen** [comment]
@dvkndn any plans of merging it? Sounds like a nice improvement...

**Comment by stephenh** [comment]
Fwiw if you like the idea of "utility class via a TypeScript DSL" but are not tied specifically to tailwinds (which is great, so np if you are), my "dropping parens" hacking has turned into this Truss framework that is very similar:

https://github.com/homebound-team/truss

(@dvkndn feel free to delete this comment if the alternative project advertising is too annoying :-) just mentioning it since it's turned out to be neat for us.)

**Comment by dvkndn** [comment]
> (@dvkndn feel free to delete this comment if the alternative project advertising is too annoying :-) just mentioning it since it's turned out to be neat for us.)

@stephenh 
What are you talking about! **Truss is great!** At a glance, I think you are solving a much more generic problem than I am, which obviously is a great thing!

Truth be told, I'm considering archiving this project since the VSCode's Tailwind extension seems doing good enough on this. If I do, I'll link to Truss for sure

**Comment by dvkndn** [comment]
And my dearest apologize for hanging this PR :( I'm no longer actively using Tailwind at work so it's hard to find time to run the PR. Thing is, dependencies of typed.tw are pretty outdated and the code itself is quite.. meta, so it's not easy to jump in. I'm sorry that I didn't see this at the beginning..

**Comment by dvkndn** [comment]
I think it's perfectly fine to merged this, but I'm having a hard time to test it all to release..
In the meanwhile, again, Truss is great! I'd try it instead of this

**Pull request merged**